lms/urls.py

```diff
@@ -13,11 +12,15 @@
 from certificates import views as certificates_views
 from config_models.views import ConfigurationModelCurrentAPIView
 from courseware.masquerade import handle_ajax as courseware_masquerade_handle_ajax
-from courseware.module_render import handle_xblock_callback, handle_xblock_callback_noauth, xblock_view, xqueue_callback
+from courseware.module_render import (
+    handle_xblock_callback_noauth,
+    XblockCallbackView, xblock_view, xqueue_callback
+)
 from courseware.views import views as courseware_views
 from courseware.views.index import CoursewareIndex
 from courseware.views.views import CourseTabView, EnrollStaffView, StaticCourseTabView
 from debug import views as debug_views
+from django_cas import views as django_cas_views
```

Comment:
Why this change? The module is conditionally included in https://github.com/open-craft/edx-platform/blob/bb828483449997a82c0b6150dfa3013eb3944701/lms/urls.py#L834, so I think we should remove this line.

Reply:
Maybe someone saw a quality warning because of the conditional import. But there other conditional imports, so I left the conditional import and removed the top one.